src/Mod/TechDraw/App/Cosmetic.cpp

```diff
@@ -980,12 +980,16 @@ std::pair<Base::Vector3d, Base::Vector3d> CenterLine::calcEndPoints(DrawViewPart
 
 bool CenterLine::Circulation(Base::Vector3d A, Base::Vector3d B, Base::Vector3d C)
 {
+    // the determinant of this matrix calculates the aread of a triangle, see
```

Comment:
Typo: should be `area of a triangle` not `aread`

Reply:
@luzpaz : in such a case, you can use suggestion feature so the author can just accept it. ;)
```suggestion
    // the determinant of this matrix calculates the area of a triangle, see
```